custom_components/tahoma/cover_devices/tahoma_cover.py

```diff
@@ -257,17 +241,6 @@ def is_closing(self):
             and current_closure.value < target_closure.value
         )
 
-    @property
-    def device_state_attributes(self):
-        """Return the device state attributes."""
-        attr = super().device_state_attributes or {}
-
-        # Obstruction Detected attribute is used by HomeKit
-        if self.has_state(IO_PRIORITY_LOCK_LEVEL_STATE):
-            attr[ATTR_OBSTRUCTION_DETECTED] = True
```

Comment:
Please don't remove this. Obstruction Detected is used by HomeKit and for now it needs to be a state attribute on the cover.

Reply:
That’s still the case ?

Reply:
https://github.com/home-assistant/core/blob/236738c455b827413ba052b1ea3bb24214d9f5db/homeassistant/components/homekit/type_covers.py#L177-L182

Reply:
Done.